network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html

```diff
@@ -87,20 +87,27 @@ <h2>{% trans 'A week later' %}</h2>
     {# Scene 4 #}
     <div class="timeline__scene tw-grid tw-grid-cols-2" data-scene="4">
       <div class="timeline__scene-left tw-grid tw-grid-cols-6 tw-grid-rows-4">
-        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-1" size="small" image='scene4/2.jpg' speed="3" text='A Dad Mistakenly Drank A Lava Lamp At Bedtime. This Is What Happened To His Kidneys.' %}
-        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4 tw-row-start-2" size="small" image='scene4/3.jpg' speed="2" text='A Student Felt A Sharp Pain In Her Side. This Is How Her Organs Shut Down.' %}
-        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-3" size="small" image='scene4/1.jpg' speed="2" text='A Mom Drank 3 Gallons Water In 2 Hours. This is What Happened to Her Brain.' %}
-        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-4" image='scene4/4.jpg' speed="3" text='A Gamer Drank 12 Energy Drinks In 10 Minutes. This Is What Happened To His Organs.' %}
+        {% trans 'A Dad Mistakenly Drank A Lava Lamp At Bedtime. This Is What Happened To His Kidneys.' as localized_title_4_2 %}
+        {% trans 'A Student Felt A Sharp Pain In Her Side. This Is How Her Organs Shut Down.' as localized_title_4_3 %}
+        {% trans 'A Mom Drank 3 Gallons Water In 2 Hours. This is What Happened to Her Brain.' as localized_title_4_1 %}
+        {% trans 'A Gamer Drank 12 Energy Drinks In 10 Minutes. This Is What Happened To His Organs.' as localized_title_4_4 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-1" size="small" image='scene4/2.jpg' speed="3" text=localized_title_4_2 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4 tw-row-start-2" size="small" image='scene4/3.jpg' speed="2" text=localized_title_4_3 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-3" size="small" image='scene4/1.jpg' speed="2" text=localized_title_4_1 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-4" image='scene4/4.jpg' speed="3" text=localized_title_4_4 %}
```

Comment:
**Question** Is there a particular reason we are not having the variable definition and it's use closer together? (trans, include, trans, include, ...)